tempesta_fw/t/functional/helpers/control.py

```diff
@@ -167,6 +167,7 @@ def form_command(self):
         threads = self.threads if self.connections > 1 else 1
         self.options.append('-t %d' % threads)
         self.options.append('-c %d' % self.connections)
+        self.options.append('--timeout 5')
```

Comment:
As discussed in chat, estimated request timeout depends on number of concurrent connections. Default of `2` is great for <100 connections, while some requests may be served more than 2 seconds with >1000 connections. But there is no straight dependency and exact required value is not clear. I believe the value must be synchronized with `server_forward_timeout` which is `60` by default.

Note that increasing serving time is not Tempesta issue, it depends on backends and wrk connections too. E.g. overall wrk performance decreases while connection count increases. 

Please, also add a comment for this change. If another benchmarker will be used in tests (TfwBomber is expected), the issue must be taken into account.